build.gradle

```diff
@@ -161,13 +173,23 @@ subprojects {
         compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: reformLogging
         compile "org.hibernate:hibernate-core:${hibernateVersion}"
         compile "org.hibernate:hibernate-entitymanager:${hibernateVersion}"
-        compile "uk.gov.hmcts.reform.auth:auth-checker-lib:${authCheckerVersion}"
+        compile (group: 'uk.gov.hmcts.reform.auth', name: 'auth-checker-lib', version: '2.1.3') {
+            // TODO remove when auth-checker-lib upgrades its dependency to spring boot 2.0.x
+            exclude group: 'uk.gov.hmcts.reform', module: 'java-logging-spring'
+        }
+        compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '1.0.4'
+        compile('org.springframework.boot:spring-boot-starter-web')
+        compile('org.springframework.boot:spring-boot-starter-data-jpa') {
+            exclude group: 'org.hibernate', module: 'hibernate-core'
+            exclude group: 'org.hibernate', module: 'hibernate-entitymanager'
+        }
         compile group: 'com.zaxxer', name: 'HikariCP', version: '3.3.1'
         compile group: 'org.jooq', name: 'jool-java-8', version: '0.9.14'
         runtime "org.elasticsearch:elasticsearch:${elasticSearchVersion}"
         compile group: 'javax.inject', name: 'javax.inject', version: '1'
 
         compile group: 'com.microsoft.azure', name: 'azure-storage', version: '8.0.0'
+        compile group: 'org.liquibase', name: 'liquibase-core', version: '3.6.3'
```

Comment:
Is this a compile dependency?

Reply:
Agreed, reverting to testCompile and removing compile dependency from the parent project